Bottom: 534afe577f2b9d47063d3db3565b7f44626ff0de
Top:    b825ac06a5414223b8694316a4190bdab860d44e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-24 16:54:47 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 8109894..a337e16 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -146,13 +146,17 @@ PCB_SRCS = \
 	hid/hidint.h 
 pcb_SOURCES = ${PCB_SRCS} core_lists.h
 
-EXTRA_pcb_SOURCES = ${DBUS_SRCS}
+EXTRA_pcb_SOURCES = ${DBUS_SRCS} ${GL_SRCS}
 DBUS_SRCS= \
 	dbus-pcbmain.c \
 	dbus-pcbmain.h \
 	dbus.h \
 	dbus.c
 
+GL_SRCS= \
+	hid/common/hidgl.c \
+	hid/common/hidgl.h
+
 BUILT_SOURCES = \
 	core_lists.h \
 	gpcb-menu.h \
@@ -267,6 +271,11 @@ BUILT_SOURCES+=	dbus-introspect.h
 
 endif
 
+# If we are building with GL support, we need some extra files
+if USE_GL
+PCB_SRCS+=	${GL_SRCS}
+endif
+
 # If we are building on win32, then compile in some icons for the
 # desktop and application toolbar
 if WIN32
